router/client/client.go

```diff
@@ -571,17 +571,14 @@ func (cl *PsqlClient) Init(tlsconfig *tls.Config) error {
 				return err
 			}
 			backend = pgproto3.NewBackend(bufio.NewReader(cl.conn), cl.conn)
-			if err != nil {
-				spqrlog.Zero.Error().Err(err).Msg("")
-				return err
-			}
 		case conn.CANCELREQ:
 			cl.csm = &pgproto3.CancelRequest{}
 			if err = cl.csm.Decode(msg); err != nil {
 				return err
 			}
 
-			return nil
+			//return nil
```

Comment:
Why?

Reply:
I don't remember exactly, probably it crashed here. I checked it rn, seems like it's working, restored changes. Will note down such things next time